pkg/managerclient/model.go

```diff
@@ -513,7 +516,18 @@ func (li TaskListItems) Render(w io.Writer) error {
 
 		var schedule string
 		if t.Schedule.Cron != "" {
-			schedule = t.Schedule.Cron
+			var cronSpec scheduler.CronSpecification
+			err := json.Unmarshal([]byte(t.Schedule.Cron), &cronSpec)
+			if err != nil {
+				schedule = t.Schedule.Cron
+			} else {
+				schedule = cronSpec.Spec
+				if cronSpec.StartDate.After(timeutc.Now()) {
+					c := scheduler.MustCron(cronSpec.Spec, cronSpec.StartDate)
+					schedule += fmt.Sprintf("; starts: %s UTC",
+						c.Next(cronSpec.StartDate).Format("2006-01-02 15:04:05"))
```

Comment:
"starts" sound a little like the task is going to be executed right on start date, but it will be executed according to crone, but not earlier than the start date. Could we change it to something like "with first activation after %s"?

The composition of start date and cron can be a little bit confusing, so could we extend sctool docs with a sentence about how they interact with each other?

Reply:
It makes sense, let me change the output to suggested "with first activation after".